ton_client/src/tests/contracts/abi_v2/Sdk.sol

```diff
@@ -0,0 +1,203 @@
+pragma ton-solidity >= 0.40.0;
```

Comment:
shouldn't it be imported by reference to githubrawcontent?

Reply:
Yep, we should remove this file and use github path to Sdk.sol in debots.